### ISSUE TITLE
changing some props in custom theme has no effect

### ISSUE THREAD

**Issue by zorx** [selemondev/nuxt-ui-vue#157]
**Describe the bug**

I'm trying to implement my own theme following the documentation. However, there are certain parameters that are not being considered, such as UButton "size," "color", "intent" etc.

But when i change UButton "rounded" for example, it works...

**To Reproduce**
```
// main
...
import nuxtLabsTheme from './customTheme'

import { createUI } from 'nuxt-ui-vue'

import App from './App.vue'

const app = createApp(App)

const UI = createUI({
  registerComponents: false,
})

app.use(UI, nuxtLabsTheme)

...
```

```
// customTheme
export default {
 ...
 UButton: {
    base : {
     ....
       default : {
         ...
         color: "red" <------------------------
       }
    }
 }
```

**Expected behavior**
the default color of a button must be red, or i still have a green button

**Comment by selemondev** [comment]
Thanks for creating an issue. I will check on it and get back to you as soon as possible 😄

**Comment by selemondev** [comment]
Have you tried customizing the color of the `UButton` component using the color prop like this:

```vue
 <UButton color="red">
      Button
 </UButton>
```

**Comment by zorx** [comment]
i know that i can change the color using the attribute, but what i want is to change the default color, and other stuff using the theme file, which seems to be impossible

**Comment by selemondev** [comment]
This issue will be resolved in the next release 😄.

**Comment by sirmathays** [comment]
Yep, I noticed the same a while ago and after looking around the code I noticed direct references to the `nuxtLabsTheme` located in the package's own files which essentially makes setting your own theme file redundant.
